pkg/scheduler/reconciliation/reconciliation.go

```diff
@@ -26,6 +34,10 @@ type Repository interface {
 	UpdateOperationState(schedulingID, correlationID string, state model.OperationState, allowInState bool, reasons ...string) error
 	WithTx(tx *db.TxConnection) (Repository, error)
 	UpdateOperationRetryID(schedulingID, correlationID, retryID string) error
+	UpdateOperationPickedUp(schedulingID, correlationID string) error
+	UpdateComponentOperationProcessingDuration(schedulingID, correlationID string, processingDuration int64) error
```

Comment:
Here, I think it would be better to have `processingDuration` of type `int` instead of `int64` as data conversion responsibility falls on the repository. Currently, it is done on the Http Handler level.